.github/workflows/merge-flow.yaml

```diff
@@ -0,0 +1,142 @@
+name: Common merge flow
+on:
+  workflow_call:
+    inputs:
+      go-version:
+        description: go version
+        required: true
+        type: string
+      upstream:
+        description: Upstream repo path in owner/repo format
+        required: true
+        type: string
+      downstream:
+        description: Downstream repo path in owner/repo format
+        required: true
+        type: string
+      downstream-branch:
+        description: Downstream branch to create PR
+        required: false
+        default: master
+        type: string
+      sandbox:
+        description: Sandbox repo path in owner/repo format. Used a base to create PR against downstream.
+        required: true
+        type: string
+      theirs:
+        description: Resets using upstream content during merge conflict.
+        required: false
+        default: ''
+        type: string
```

Comment:
Should we use a boolean type here?

Reply:
This actually takes a list of files to be reset using upstream contents.

Reply:
Ok that makes sense. Can we just adjust the description to reflect that more explicitly?